### PR TITLE
Link and formatting fixes

### PR DESCRIPTION
**Pull request by LisaDelaney**

### CODE REVIEW

File: docs/conceptual/using-gpu-sanitizer.md [hunk 1]
```diff
@@ -235,7 +235,7 @@ $ rocgdb <path to application>
 
 ### Using ASan with a short HIP application
 
-Refer to the following example to use address sanitizer with a short HIP application,
+Refer to the following example to use ASan with a short HIP application,
```
Comment (Rmalavally) [inline docs/conceptual/using-gpu-sanitizer.md:238]: @LisaDelaney - ASAN (all uppercase?)

Comment (Rmalavally) [reply docs/conceptual/using-gpu-sanitizer.md:238]: Looks good, Lisa. 

Did we put the abbreviation in parenthesis in the first instance? For example, AddressSanitizer (ASan). 
I noticed that LLVM recommends camel case, so we should retain the guideline for consistency. 

Thanks for your response.

Comment (LisaDelaney) [reply docs/conceptual/using-gpu-sanitizer.md:238]: @Rmalavally - Yep, it's also in parentheses in the title, which I have mixed feelings about. This convention is not consistent across the docs, so probably warrants a conversation at some point as to whether we want to do this or not.

Comment (Rmalavally) [reply docs/conceptual/using-gpu-sanitizer.md:238]: > @Rmalavally - Yep, it's also in parentheses in the title, which I have mixed feelings about. This convention is not consistent across the docs, so probably warrants a conversation at some point as to whether we want to do this or not.

For sure. Let's discuss.

File: CHANGELOG.md [hunk 2]
```diff
@@ -434,54 +435,54 @@ In csritlu0, the algorithm rocsparse_itilu0_alg_sync_split_fusion has some accur
...
+* Fixed issue where `transform_iterator` would not compile with `__device__`-only operators.
 
 ##### Changed
 
-- Updated `docs` directory structure to match the standard of [rocm-docs-core](https://github.com/RadeonOpenCompute/rocm-docs-core).
-- Removed references to and workarounds for deprecated hcc
+* Updated `docs` directory structure to match the standard of [rocm-docs-core](https://github.com/RadeonOpenCompute/rocm-docs-core).
+* Removed references to and workarounds for deprecated hcc
 
 #### rocWMMA 1.2.0
```
Comment (samjwu) [inline CHANGELOG.md:448]: Parts of the CHANGELOG.md are generated from other projects

eg: for rocWMMA https://github.com/ROCmSoftwarePlatform/rocWMMA/blob/release/rocm-rel-5.7/CHANGELOG.md

**Review by saadrahim — DISMISSED** [review]

**Review by saadrahim — APPROVED** [review]

**Pull request merged**